tools/walletextension/wallet_extension.go

```diff
@@ -214,12 +223,6 @@ func logAndSendErr(resp http.ResponseWriter, msg string) {
 	http.Error(resp, msg, httpCodeErr)
 }
 
-// ViewingKey is the packet of data sent to the enclave when storing a new viewing key.
-type ViewingKey struct {
```

Comment:
This structure was only relevant when shoving the viewing key down a channel to Obscuro facade.